### ISSUE TITLE
Autoruns column width problem。。。

### ISSUE THREAD

**Issue by i5151** [MicrosoftDocs/feedback#964]
![FaWXBF.jpg](https://s1.ax1x.com/2018/12/16/FaWXBF.jpg)
OS: Windows 10 1809 x64,
Display resolution: 3840X2160,
Autoruns version: 13.93
I adjust the width of each column and return to the original width after restarting. How can I make Autoruns remember the width after I adjusted?

**Comment by Powerhelmsman** [comment]
@i5151 What's content page you refer to?

**Comment by i5151** [comment]
Any page in Autoruns is the same.
I adjusted the width.
![F0L1it.png](https://s1.ax1x.com/2018/12/18/F0L1it.png)
After restarting the computer, open Autoruns again, and it will look like this:
![F0LQII.png](https://s1.ax1x.com/2018/12/18/F0LQII.png)
I have looked at the key values of the registry. After I adjusted the width, the key values did change. But starting again, Autoruns will reset the key value to the initial state.
